torch/testing/_internal/opinfo/core.py

```diff
@@ -2687,13 +2687,6 @@ def __init__(
             torch_ref_method,
             torch_ref_inplace,
         ) = get_foreach_method_names(name)
-        if name == "zero":
-            # note(crcrpar): `foreach_method` for `"zero"` is `None` but `None` would call
-            # `_getattr_qual` in `OpInfo.__post_init__` which should fail since `_foreach_zero`
-            # is not defined at the moment. Thus to skip the qualification, set a similar torch
-            # function.
-            assert foreach_method is None
-            foreach_method = torch.zero_
```

Comment:
`zero`-related changes in core are to support this test. Might split it out or skip.